docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+
+```javascript
+	module.exports = {
+	    consumer_key: process.env.consumer_key,
+	    consumer_secret: process.env.consumer_secret,
+	    access_token: process.env.access_token,
+	    access_token_secret: process.env.access_token_secret,
+	};
+```
+
+After all this, open up `index.js`, or whatever you've named it, and get ready to type some javascript
+
+## Code It Up.
+
+```javascript
+	console.log("SNES Soundtracks booting up");
+
+	//making sure npm run develop works
+	if (process.env.NODE_ENV === "develop") {
+	    require("dotenv").config();
+	};
+
+	//rules for node-schedule
+	var schedule = require("node-schedule");
+	var rule = new schedule.RecurrenceRule();
+	  rule.dayOfWeek = 1,
+	  rule.hour = 10;
+	  rule.tz = "Etc/GMT+4";
+
+	//array to pull soundtracks from
+	var soundtrackArray = [ "an array of youtube video URLs"];
+	var soundtrackArrayLength = soundtrackArray.length;
+	var soundtrackArrayElement = Math.floor(Math.random() * soundtrackArrayLength);
+
+```
+</br>
+
+At the beginning, I log a start up message to the console just so I know it’s running. Next is an if statement to use the dotenv when the node environment is ‘develop’, which is handled in the scripts of the json file from earlier. We set up a few variables for the node-schedule so the bot can tweet on a set day and time. I pulled this directly from the docs for node-schedule. Basically, it will tweet every Monday at 10 am Eastern Daylight Savings Time. Finally, I set up an array for the bot to pull from at random with the last three lines of this section. I removed the URLs for the tutorial.
+
+```javascript
+	// Create a Twitter object to connect to Twitter API
+	var Twit = require('twit');
+
+	// Pulling keys from another file
+	var config = require('./config.js');
+	// Making a Twit object for connection to the API
+	var T = new Twit(config);
+
+	// Setting up a user stream
+	var stream = T.stream('statuses/filter', { track: '@SnesSoundtracks' });
+
+	// Now looking for tweet events
+	// See: https://dev.Twitter.com/streaming/userstreams
+	stream.on('tweet', pressStart);
+```
+</br>
+
+This is where we start using Twit. We create an object called `Twit` that requires the npm, and then pass a configure that requires the configuration file into the object. We then use new to create `T`, an instance of the object from before. From now on, when we want to use something from Twit, we simply use T.whatever in order to call up the property, field, or method we need from their library. We set up a stream so that we are monitoring specifically @SnesSoundtracks while the code is running. Finally, we create an event listener with `stream.on`, and use a string parameter to name it, and input a function we’ve called, `pressStart`. `pressStart` is defined in the next set of code.
+
```

Comment:
I moved the instructions for creating the `config.js` file into this part of the guide, because it felt like it made sense to put it near the `snes.js` code that references it.